source/Nabla/COpenGLBuffer.h

```diff
@@ -73,12 +54,13 @@ class COpenGLBuffer final : public IGPUBuffer, public IDriverMemoryAllocation
         inline bool canUpdateSubRange() const override {return cachedFlags&GL_DYNAMIC_STORAGE_BIT;}

```

Comment:
you can get rid of this in all buffer classes

Reply:
isnt it kinda useful?

Reply:
does anything check `canUpdateSubRange` ?

Reply:
i dont think so, but not 100% sure

Reply:
its just useful to user so that he can check if he can do cmdbuf->updateBuffer() with the buffer

Reply:
ah yeah I forgot about that commandbuffer can do small updates.

but does Vulkan have some sort of a flag on a buffer which says if you can do `vkCmdUpdateBuffer` or not?